docker-compose.playwright.yaml

```diff
@@ -4,9 +4,9 @@
 #
 # This file comes from https://github.com/julienloizelet/ddev-playwright
 #
-version: '3.6'
```

Comment:
The `version` hasn't been used or required by docker-compose for a couple of years at least.